src/test/java/test/LoginTest.java

```diff
@@ -30,8 +33,9 @@ public void setup() throws InterruptedException{
     @Test
     public void loginTest() {
         homepage = new HomePage(driver);
-        homepage.clickOnSignInButton()
-                .setEmailAddressField("razvanratiu@yahoo.com")
+        loginPage = new LogInPage(driver);
+        homepage.clickOnSignInButton();
+        loginPage.setEmailAddressField("razvanratiu@yahoo.com")
                 .setPasswordField("Softvision10")
                 .clickonLogInbutton();
         WebElement myAccountMessage = driver.findElement(By.cssSelector(".info-account"));
```

Comment:
Based on the above comment. Update the rest of the test to correctly navigate from a page to another through objects.